packages/nextjs/src/pages/ClerkProvider.tsx

```diff
@@ -9,11 +9,9 @@ import { invalidateNextRouterCache } from './invalidateNextRouterCache';
 // TODO: Import from shared once [JS-118] is done
 const useSafeLayoutEffect = typeof window !== 'undefined' ? React.useLayoutEffect : React.useEffect;
 
-__internal__setErrorThrowerOptions({
-  packageName: '@clerk/nextjs',
-});
+__internal__setErrorThrowerOptions({ packageName: '@clerk/nextjs' });
 
-export * from '@clerk/clerk-react';
+// export * from '@clerk/clerk-react';
```

Comment:
Should we delete this line?

Reply:
https://github.com/clerkinc/javascript/pull/1104#discussion_r1178943340